app/controllers/garments_controller.rb

```diff
@@ -1,15 +1,14 @@
 class GarmentsController < ApplicationController
 
   def new
-
+    @garment = Garment.new
```

Comment:
there should be a conflict here if you merged with master, can you first merge with master and resolve the conflicts, if any?